doc/thing/thing-requirements.rst

```diff
@@ -0,0 +1,147 @@
+Requirements
+============
+
+Linux based OS
+--------------
+
+The actual version of KNoT Zephyr SDK is only available for Linux based systems.
+You may consider using a Virtual Machine running a Linux distribution on other systems.
+
+Zephyr
+------
+
+KNoT uses a fork of Zephyr repository.
+
+- Set up a Zephyr development environment by following these `instructions <https://docs.zephyrproject.org/latest/getting_started/index.html#set-up-a-development-system>`_.
+- Program the ``$HOME/.profile`` to always set **ZEPHYR_TOOLCHAIN_VARIANT** and **ZEPHYR_SDK_INSTALL_DIR**.
+
+.. code-block:: bash
+
+   $ echo "export ZEPHYR_TOOLCHAIN_VARIANT=zephyr" >> $HOME/.profile
+   $ echo "export ZEPHYR_SDK_INSTALL_DIR=`readlink -f <SDK-INSTALLATION-DIRECTORY>`" >> $HOME/.profile
+
+.. note:: Replace ``<SDK-INSTALLATION-DIRECTORY>`` by the actual Zephyr SDK install path
+
+- Install the west binary and bootstrapper
+
+.. code-block:: bash
+
+   $ pip3 install --user west
+
+- Create an enclosing folder and change directory
+
+.. code-block:: bash
+
+   $ mkdir zephyrproject
+   $ cd zephyrproject
+
+- Clone KNoT Zephyr fork
+
+.. code-block:: bash
+
+   $ git clone -b zephyr-knot-v1.14.0 https://github.com/CESARBR/zephyr.git
+
+- Initialize west. Inside zephyrproject directory run:
+
+.. code-block:: bash
+
+   $ west init -l zephyr/
+   $ west update
+
+.. note:: If the system can't find west, try logging out and in again.
+
+- Set up zephyr environment variables
+
+.. code-block:: bash
+
+   $ source zephyr/zephyr-env.sh
+
+- Program the ``$HOME/.profile`` to always source zephyr-env.sh when you log in.
+
+.. code-block:: bash
+
+   $ echo "source $ZEPHYR_BASE/zephyr-env.sh" >> $HOME/.profile
+
+.. note:: If you skip this step, it will be necessary to manually source zephyr-env.sh every time a new terminal is opened.
+
+
+nRF5x Command Line Tools
```

Comment:
You might merge the `nRF5x Command Line Tools` and `Install Segger JLink` sections by telling the user to download [the nRF Command Line Tools package](https://www.nordicsemi.com/-/media/Software-and-other-downloads/Desktop-software/nRF-command-line-tools/sw/Versions-10-x-x/nRFCommandLineTools1021Linuxamd64tar.gz) and install the `JLink_Linux_V644e_x86_64.deb` and `nRF-Command-Line-Tools_10_2_1_Linux-amd64.deb` files.

You can find the commands between https://github.com/CESARBR/zephyr-knot-sdk/blob/master/docker/Dockerfile#L126 and https://github.com/CESARBR/zephyr-knot-sdk/blob/master/docker/Dockerfile#L138

Reply:
Done